wscl-issues/proposed/pprint-invalid-kind-or-relative-to

```diff
@@ -0,0 +1,172 @@
+Issue:          PPRINT-INVALID-KIND-OR-RELATIVE-TO
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         proposed
+Edit History:   03-Oct-21, Version 1 by Tarn W. Burton
+References:     PPRINT-INDENT, PPRINT-NEWLINE, PPRINT-TAB
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the Exceptional
+  Situations section in the description of the function
+  PPRINT-INDENT, the function PPRINT-NEWLINE, and the function
+  PPRINT-TAB are not consistent in regards to the errors signaled
+  for invalid KIND or RELATIVE-TO arguments. The text for each
+  function is listed below.
+
+  1. PPRINT-INDENT: "An error is signaled if relative-to is any
+     object other than :block or :current."
+
+  2. PPRINT-NEWLINE: "An error of type type-error is signaled if
+     kind is not one of :linear, :fill, :miser, or :mandatory."
+
+  3. PPRINT-TAB: "An error is signaled if kind is not one of
+     :line, :section, :line-relative, or :section-relative."
+
+Proposal (PPRINT-INVALID-KIND-OR-RELATIVE-TO:SIGNAL-ERROR-IN-SAFE-CODE):
+
+  This proposal changes the description of the PPRINT-INDENT and
+  the PPRINT-TAB functions to specify that a TYPE-ERROR will be
+  signaled in these situations. The individual proposed changes
+  to the descriptions are:
+
+  1. Change the text of section "Exceptional Situations" of the
+     PPRINT-INDECT function to read: "An error of type type-error
+     is signaled if relative-to is any object other than :block
+     or :current."
+
+  2. Change the text of section "Exceptional Situations" of the
+     PPRINT-TAB function to read: "An error of type type-error
+     is signaled if kind is not one of :line, :section,
+     :line-relative, or :section-relative."
+
+Test Cases:
+
+  (defun one (object)
+    (declare (optimize (safety 3)))
+    (pprint-indent object 0))
+
+  (defun two (object)
+    (declare (optimize (safety 3)))
+    (pprint-newline object))
+
+  (defun three (object)
+    (declare (optimize (safety 3)))
+    (pprint-tab object 10 10))
+
+  (one :bar) => [signals TYPE-ERROR with expected type
+                  (MEMBER :BLOCK :CURRENT)]
+  (two :bar) => [signals TYPE-ERROR with expected type
+                  (MEMBER :LINEAR :MISER :FILL :MANDATORY)]
+  (three :bar) => [signals TYPE-ERROR with expected type
+                    (MEMBER :LINE :SECTION :LINE-RELATIVE
+                            :SECTION-RELATIVE)]
+
+Rationale:
+
+  We think it was a small error in the draft ANSI specification
+  to not specify that the PPRINT-INDENT and PPRINT-TAB
```

Comment:
Maybe "specify that the *functions* PPRINT-INDENT …"?

Reply:
Fixed.